src/DynamoPublish/Properties/Resources.resx

```diff
@@ -180,6 +180,9 @@
   <data name="ServerNotFoundMessage" xml:space="preserve">
     <value>Server not found.</value>
   </data>
+  <data name="InvalidNodeMessage" xml:space="preserve">
+    <value>There are not allowed nodes to publish your customizer: </value>
```

Comment:
"Your workspace contains nodes that are currently not allowed in a customizer: "